src/app/component/home/components/eco-events/eco-events.component.html

```diff
@@ -2,7 +2,7 @@
   <div id="eco-events-wrapper" *ngIf="latestNews.length > 0">
```

Comment:
What is the reason to use 'id' in your elements?

Reply:
> What is the reason to use 'id' in your elements?

Fixed